src/index.js

```diff
@@ -1 +1,27 @@
-import './assets/stylesheets/style.css';
\ No newline at end of file
+import './assets/stylesheets/style.css';
+import buildList from './modules/buildList.js';
+import createGame from './modules/createGame.js';
+import getScores from './modules/getScores.js';
+import submitScores from './modules/submitScores.js';
+
+const submitBtn = document.querySelector('.submit-btn');
+const refreshBtn = document.querySelector('.refresh-btn');
+const nameInput = document.querySelector('.player-name');
+const scoreInput = document.querySelector('.player-score');
+
+const container = document.querySelector('.scores-list');
+container.style.display = 'none';
+
+window.addEventListener('load', async () => {
+  let { result } = await createGame();
+  result = result.substring(14, 34);
+
+  submitBtn.addEventListener('click', async () => {
+    await submitScores(nameInput.value, scoreInput.value, `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${result}/scores`);
```

Comment:
- [ ] After you make the API fetch response by the submitScore function it is good if you clean the input fields. Kindly add a couple of lines here to clean the name and score input fields. Simply assigning an empty string to the nameInput and scoreInput would solve it. :pray: